$OpenBSD: patch-cmake_modules_FindQt_cmake,v 1.1.1.1 2013/05/09 19:25:59 zhuk Exp $
Unbreak configure stage.
Telepathy-qt devs use customized version of module bundled with CMake,
so we cannot just zap this module.
--- cmake/modules/FindQt.cmake.orig	Wed May  8 16:12:11 2013
+++ cmake/modules/FindQt.cmake	Wed May  8 16:12:34 2013
@@ -8,7 +8,7 @@
 # For details see the accompanying COPYING-CMAKE-SCRIPTS file.
 
 IF(NOT QT_QMAKE_EXECUTABLE)
-  FIND_PROGRAM(QT_QMAKE_EXECUTABLE_FINDQT NAMES qmake qmake4 qmake-qt4 qmake5 qmake-qt5
+  FIND_PROGRAM(QT_QMAKE_EXECUTABLE_FINDQT NAMES qmake4 qmake-qt4 qmake5 qmake-qt5 qmake
                PATHS "${QT_SEARCH_PATH}/bin" "$ENV{QTDIR}/bin")
   SET(QT_QMAKE_EXECUTABLE ${QT_QMAKE_EXECUTABLE_FINDQT} CACHE PATH "Qt qmake program.")
 ENDIF(NOT QT_QMAKE_EXECUTABLE)
